### PR TITLE
cri-containerd: acquire cri_containerd version in faster way

### PR DESCRIPTION
**Pull request by Pennyzct**
To acquire version of `cri_containerd`, you only need the file `vendor.conf` under `containerd/containerd` repo. 
You don't need to download the whole repo, that's time-consuming.

### CODE REVIEW

File: .ci/install_cri_containerd.sh [hunk 1]
```diff
@@ -31,12 +30,8 @@ cri_containerd_tarball_version=$(get_version "externals.cri-containerd.version")
 cri_containerd_repo=$(get_version "externals.cri-containerd.url")
 
 echo "Get cri_containerd version"
-go get "${containerd_repository}"
-cri_containerd_version=$(
-	cd "${GOPATH}/src/${containerd_repository}";
-	git checkout "v${cri_containerd_tarball_version}" >&2;
-	cat vendor.conf | grep "github.com/containerd/cri" | awk '{print $2}';
-)
+cri_containerd_version_url="https://raw.githubusercontent.com/containerd/containerd/v${cri_containerd_tarball_version}/vendor.conf"
```
Comment (grahamwhaley) [inline .ci/install_cri_containerd.sh]: heh, well, I reckon @jodh-intel would ask you to move this const nearer the top of the file, like it used to be ;-)

**Comment by grahamwhaley** [comment]
I've wiggled the WIP label on/off, as that should re-trigger the zuul WIP check (which cannot be re-triggered with the zuul phrase - that only triggers the SoB check ;-) )

File: .ci/install_cri_containerd.sh (hunk 1 above)
Comment (Pennyzct) [reply .ci/install_cri_containerd.sh]: 23333, updated. ;).

Comment (Pennyzct) [reply .ci/install_cri_containerd.sh]: Hi @grahamwhaley I suddenly found that I couldn't move this const nearer the top of the file, it includes another variable `cri_containerd_tarball_version`.
that's why majority of CI failed. 😭
```
11:13:56 /tmp/jenkins/workspace/kata-containers-tests-ubuntu-18-04-PR/go/src/github.com/kata-containers/tests/integration/containerd/shimv2/../../../.ci/install_cri_containerd.sh: line 15: cri_containerd_tarball_version: unbound variable
```

**Comment by Pennyzct** [comment]
Hi guys
I almost got all green here. I think Power8's failure is not irrelevant with this PR~~~ ;).

**Comment by chavafg** [comment]
yeah, Power8 job has been failing since the beginning of the week I think, 

any idea @nitkon? 
Seems like we are not able to install docker 18.06 with the [`manage_ctr_mgr.sh`](https://github.com/kata-containers/tests/blob/master/cmd/container-manager/manage_ctr_mgr.sh) script

```
04:35:02 <13>Aug  8 15:05:02 manage_ctr_mgr.sh: Installing docker v18.06-ce
04:35:02 Reading package lists...
04:35:02 Building dependency tree...
04:35:03 Reading state information...
04:35:03 software-properties-common is already the newest version (0.96.20.8).
04:35:03 apt-transport-https is already the newest version (1.8.2).
04:35:03 ca-certificates is already the newest version (20190110).
04:35:03 The following packages were automatically installed and are no longer required:
04:35:03   aufs-tools cgroupfs-mount conntrack cri-tools ebtables
04:35:03   linux-headers-4.15.0-29 linux-headers-4.15.0-29-generic
04:35:03   linux-image-unsigned-4.15.0-29-generic linux-modules-4.15.0-29-generic pigz
04:35:03   socat
04:35:03 Use 'sudo apt autoremove' to remove them.
04:35:03 0 upgraded, 0 newly installed, 0 to remove and 598 not upgraded.
04:35:03 Warning: apt-key output should not be parsed (stdout is not a terminal)
04:35:04 OK
04:35:17 Hit:1 https://download.docker.com/linux/ubuntu xenial InRelease
04:35:17 Hit:2 http://us.ports.ubuntu.com/ubuntu-ports xenial InRelease
04:35:17 Hit:3 http://us.ports.ubuntu.com/ubuntu-ports xenial-updates InRelease
04:35:17 Hit:4 http://us.ports.ubuntu.com/ubuntu-ports xenial-backports InRelease
04:35:17 Hit:6 http://ppa.launchpad.net/alexlarsson/flatpak/ubuntu xenial InRelease
04:35:17 Hit:7 http://ports.ubuntu.com/ubuntu-ports xenial-security InRelease
04:35:17 Hit:8 http://ftp.de.debian.org/debian sid InRelease
04:35:17 Hit:5 https://packages.cloud.google.com/apt kubernetes-xenial-unstable InRelease
04:35:18 Reading package lists...
04:35:20 Reading package lists...
04:35:20 Building dependency tree...
04:35:20 Reading state information...
04:35:20 E: Version '' for 'docker-ce' was not found
```

**Pull request merged**